test/test.js

```diff
@@ -79,5 +80,24 @@ describe('Serializer', () => {
 
       strictEqual(toCanonical(result), toCanonical(quads))
     })
+
+    describe('should generate module when the type is', () => {
+      [undefined, 'commonjs', 'esm', 'ts'].forEach(module => {
+        it(module || 'undefined', function () {
+          const quads = [
+            rdf.quad(
+              rdf.blankNode(),
+              rdf.namedNode('http://example.org/predicate1'),
+              rdf.literal('object')
+            )
+          ]
+
+          const serializer = new Serializer({ module })
+          const code = serializer.transform(quads)
+
+          expect(code).to.matchSnapshot(this)
```

Comment:
Snapshot tests are testing many times to much. I think this test should also just search for the line of the export and ignore the rest.